app/visualizations.py

```diff
@@ -5,25 +5,30 @@
 from app.db_ops import get_judge_df, get_table, get_df
 import pandas as pd
 
-# def get_judge_side_bar(judge_name: str) -> go.Figure:
-#         """
-#         Function grabs data from database by judge and creates a
-#         side-by-side bar chart for visualization.
-#         Input: judge_name (name of judge to filter by)
-#         Output: fig (object with visualization data)
-#         """
-#         df = get_judge_df(judge_name)
-#         df["count"] = 1
-#         grouped_df = df.groupby(by=["protected_grounds",
-#                                     "outcome"]).agg({
-#                                                     "count": ["sum"]
-#                                                     }).reset_index()
-#         fig = px.bar(grouped_df, x="protected_grounds", y="count",
-#                      color="outcome",
-#                      color_discrete_map={"granted": '#00D100',
-#                                          "denied": "#D10000"},
-#                      barmode="group", title="Side-by-Side Bar Chart")
-#         return fig
+'''
+def get_judge_side_bar(judge_name: str) -> go.Figure:
+        """
+        Function grabs data from database by judge and creates a
+        side-by-side bar chart for visualization.
+        Input: judge_name (name of judge to filter by)
+        Output: fig (object with visualization data)
+        """
+        df = get_judge_df(judge_name)
+        df["count"] = 1
+        grouped_df = df.groupby(by=["protected_grounds",
+                                    "outcome"]).agg({
+                                                    "count": ["sum"]
+                                                    }).reset_index()
+        fig = px.bar(grouped_df, x="protected_grounds", y="count",
+                     color="outcome",
+                     color_discrete_map={"granted": '#00D100',
+                                         "denied": "#D10000"},
+                     barmode="group", title="Side-by-Side Bar Chart")
+        return fig
+
+# The function above no longer works and was replaced by the one below
```

Comment:
If this function no longer works, is there a reason its still here? Other than that, it looks solid!

Reply:
Since we weren't able to get it fully working deployed, I wanted to keep it in there in case somebody down the line needs it as reference. It may be worth deleting for clarity sake, but my thought was that we should make sure the other functions/endpoints were working correctly before just completely removing.